tests/otel_integrations/test_aiohttp_client.py

```diff
@@ -288,6 +288,197 @@ async def test_aiohttp_client_exception_handling_with_hooks(exporter: TestExport
     )
 
 
+@pytest.mark.anyio
+async def test_aiohttp_client_capture_response_body(exporter: TestExporter, test_app: aiohttp.web.Application):
+    """Test that aiohttp client captures response body and headers when configured to do so."""
+
+    try:
+
+        async def handler(request: aiohttp.web.Request) -> aiohttp.web.Response:
+            return aiohttp.web.json_response({'good': 'response'})
+
+        test_app.router.add_get('/body', handler)
+
+        async with aiohttp.test_utils.TestServer(test_app) as server:
+            await server.start_server()
+
+            logfire.instrument_aiohttp_client(capture_headers=True, capture_response_body=True)
+
+            async with aiohttp.ClientSession() as session:
+                async with session.get(f'http://localhost:{server.port}/body') as response:  # type: ignore
+                    await response.json()
+                    await response.read()
```

Comment:
please assert what these return

Reply:
Done, we see the response in the snapshot as well though so what's the reason to add another assert here?